app/src/main/java/com/gathering/android/profile/ProfileFragment.kt

```diff
@@ -4,19 +4,100 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.core.os.bundleOf
 import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.gathering.android.R
+import com.gathering.android.common.ImageLoader
+import com.gathering.android.common.getNavigationResultLiveData
 import com.gathering.android.databinding.FrgProfileBinding
+import com.gathering.android.event.KEY_ARGUMENT_UPDATE_USER_INFO_DATA
+import com.gathering.android.profile.updateinfo.UpdatedUserInfo
+import dagger.hilt.android.AndroidEntryPoint
+import javax.inject.Inject
 
+@AndroidEntryPoint
 class ProfileFragment : Fragment() {
 
     private lateinit var binding: FrgProfileBinding
 
+    @Inject
+    lateinit var viewModel: ProfileViewModel
+
+    @Inject
+    lateinit var imageLoader: ImageLoader
+
+    private var photoUrl: String = ""
```

Comment:
Please remove these values, there is no need to save these values in the fragment body.

Reply:
it's solved